configure.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+echo "Running configuration script"
+export DEBIAN_FRONTEND=noninteractive
+
+echo "US/Central" | sudo tee /etc/timezone
+dpkg-reconfigure --frontend noninteractive tzdata
+
+echo "Add custom sources"
+# Add ansible key
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 93C4A3FD7BB9C367
```

Comment:
I still prefer full fingerprints, but sixteen hex characters is probably okay :-).

Reply:
Oh nuts I meant to address that thanks for the reminder

Reply:
6125E2A8C77F2818FB7BD15B93C4A3FD7BB9C367

I looked it up here: https://launchpad.net/~ansible/+archive/ubuntu/ansible but realized there's probably a handy one line command in your brain for getting the full key right from cli.

Would love to know it for future reference if you have that handy.